src/experimental/createExperimentalInk.tsx

```diff
@@ -0,0 +1,164 @@
+import ansiEscapes from 'ansi-escapes';
+import isCI from 'is-ci';
+import throttle from 'lodash.throttle';
+import logUpdate from 'log-update';
+import React, {ReactNode} from 'react';
+import signalExit from 'signal-exit';
+import {App} from '../components/App';
+import {Ink, InkOptions} from '../ink';
+import {instances} from '../instances';
+import {reconciler} from './reconciler';
+import {createRenderer} from './renderer';
+import {createNode} from './dom';
+
+export function createExperimentalInk(options: InkOptions): Ink {
```

Comment:
What's the reason here for not keeping the `Instance` class, in favor of a function?

Reply:
I was learning about how Ink works and refactoring to TypeScript at the same time. It was easier for me to trust a closure with internal state than a mutable object. Now that I know how Ink works, I can revert back to class/instance if you prefer. Please let me know.

Reply:
Yes please, I think it makes more sense to have `Instance` class as it was before.

Reply:
Ok, I'll work on this later today.

Reply:
Ok, this is done. I called the class Ink instead of Instance. I left Instance type as result of render so the API doesn't change.